clients/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scale-llm-engine"
-version = "0.0.0.beta34"
+version = "0.0.0.beta35"
```

Comment:
there should be two other places where this version bump needs to happen (for a total of 3 places)